moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -134,10 +134,7 @@ class Servo
 private:
   /**
    * \brief Convert a give twist command to planning frame
-   * This implementation does not differentiate between body twist and spatial twist,
-   * i.e if the command is as seen from a stationary or non-stationary frame.
-   * The result of this transformation will only be accurate if the command frame is a stationary frame.
-   * See issue: https://github.com/ros-planning/moveit2/issues/2150
+   * The result of this transformation will be accurate only if the command frame is a stationary frame or end-effector frame.
```

Comment:
Should we be linking here to some of the resources in @AndyZe's Eigen conversion function in case people want to read more about that?

Also, I'm not sure that "not accurate" is quite the right thing here anymore. It really is whether you want the twist to be applied about the planning frame vs. about the end effector frame, which is what caused all our confusion last week in the first place. So probably worth rewording that here as well?

Reply:
Yeah that is a good idea.

Reply:
> It really is whether you want the twist to be applied about the planning frame vs. about the end effector frame,

The "not accurate" part is about the command being in some moving frame other than ee frame. (eg: the case of elbow frame)